assets/css/main.scss

```diff
@@ -84,3 +84,23 @@
 .btn {
     background-color: #cecece;
 }
+
+blockquote: {
+    background: #f4f4f4;
+    border: 1px solid #ddd;
+    border-left-color: rgb(221, 221, 221);
+    border-left-style: solid;
+    border-left-width: 1px;
+    border-left: 3px solid #008487;
```

Comment:
These lines can be removed:
border-left-color: rgb(221, 221, 221);
border-left-style: solid;
border-left-width: 1px;

Because you overwrite them by the next line:
border-left: 3px solid #008487;

Reply:
Ok, thanks